latiss/transmission_filter/sdssg_65mm~empty/20221005T000000.ecsv

```diff
@@ -0,0 +1,250 @@
+# %ECSV 0.9
+# ---
+# datatype:
+# - {name: wavelength, unit: nm, datatype: float32}
+# - {name: throughput, unit: '', datatype: float32}
```

Comment:
Files below use `-` for the dimensionless value, but this one uses `''` here and on L22. Does that matter?

Reply:
astropy doesn't like anything for dimensionless units (as far as I can tell), so I think these are equivalent.

Reply:
That's fine, just thought they should all agree. I'm sure it doesn't actually matter though.